docs/structure-samples/javascript/javascript.md

```diff
@@ -0,0 +1,75 @@
+## JavaScript project folder structure
+
+The following guidelines and the folders included, represent the conventional structure for a JavaScript or TypeScript standard application, as an npm package.
+
+```bash
+
+~root
+│
+├── .devcontainer.json              
+├── azure.yaml                      
+│
+├── .github/
+│   └── workflows/
+│       ├── workflow1.yml           
+│       ├── workflow2.yml           
+│       └── ...                     
+│
+├── infra/
+│   ├── main.bicep                  
+│   ├── main.parameters.bicep       
+│   ├── abbreviations.json          
+│   └── ...                         
+│
+├── src/                            
```

Comment:
Maybe we should also suggest the `packages/<component>` alternative in case of a NPM monorepo?

Reply:
added